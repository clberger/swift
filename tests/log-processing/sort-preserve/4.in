bar 3 III
foo 1 I
foo 5 AAAAA
foo 5 CCCCC
foo 5 BBBBB
bar 10 X
bar 6 VI
foo 4 IV2
foo 4 IV1
foo 7 VII
foo 5 EEEEE
foo 5 DDDDD
foo 2 II
foo 9 IX
qux 8 VIII
fol 11 XI
