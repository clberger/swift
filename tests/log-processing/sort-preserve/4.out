bar 3 III
bar 6 VI
bar 10 X
fol 11 XI
foo 1 I
foo 2 II
foo 4 IV2
foo 4 IV1
foo 5 AAAAA
foo 5 CCCCC
foo 5 BBBBB
foo 5 EEEEE
foo 5 DDDDD
foo 7 VII
foo 9 IX
qux 8 VIII
